<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AfterLogin</class>
 <widget class="QWidget" name="AfterLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(243, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_afterLoginImage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_afterLoginName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_changeHeaderimage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton_changeHeaderimage
{
	border: 2px solid #4CAF50;
	border-radius: 10px;
	background-color: white;
}
QPushButton#pushButton_changeHeaderimage:hover
{
	background-color: #E8F5E9;
}
QPushButton#pushButton_changeHeaderimage:pressed
{
	background-color: #C8E6C9;
}
</string>
   </property>
   <property name="text">
    <string>更改头像</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>280</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton_back
{
	border: 2px solid #4CAF50;
	border-radius: 10px;
	background-color: white;
}
QPushButton#pushButton_back:hover
{
	background-color: #E8F5E9;
}
QPushButton#pushButton_back:pressed
{
	background-color: #C8E6C9;
}</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nickname">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_enterMusicplayer">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton_enterMusicplayer {  
    border: none; /* 或者完全省略这行，因为没有设置边框宽度和样式，它默认为none */  
    border-radius: 10px; /* 设置圆角边框 */  
    background-color: rgba(255, 255, 255, 0); /* 设置为透明背景 */  
    color: black; /* 文字颜色 */  
}  
  
QPushButton#pushButton_enterMusicplayer:hover {  
    background-color: rgba(200, 255, 200, 0.5); /* 悬停时浅浅绿色 */  
}  
  
QPushButton#pushButton_enterMusicplayer:pressed {  
    background-color: rgba(0, 255, 0, 0.5); /* 按下时亮绿色 */  
}</string>
   </property>
   <property name="text">
    <string>本地和下载</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_AIchat">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/AI_chat.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_AIchat">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton_AIchat {  
    border: none; /* 或者完全省略这行，因为没有设置边框宽度和样式，它默认为none */  
    border-radius: 10px; /* 设置圆角边框 */  
    background-color: rgba(255, 255, 255, 0); /* 设置为透明背景 */  
    color: black; /* 文字颜色 */  
}  
  
QPushButton#pushButton_AIchat:hover {  
    background-color: rgba(200, 255, 200, 0.5); /* 悬停时浅浅绿色 */  
}  
  
QPushButton#pushButton_AIchat:pressed {  
    background-color: rgba(0, 255, 0, 0.5); /* 按下时亮绿色 */  
}</string>
   </property>
   <property name="text">
    <string>AI助手</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_onlinemusic">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton_onlinemusic {  
    border: none; /* 或者完全省略这行，因为没有设置边框宽度和样式，它默认为none */  
    border-radius: 10px; /* 设置圆角边框 */  
    background-color: rgba(255, 255, 255, 0); /* 设置为透明背景 */  
    color: black; /* 文字颜色 */  
}  
  
QPushButton#pushButton_onlinemusic:hover {  
    background-color: rgba(200, 255, 200, 0.5); /* 悬停时浅浅绿色 */  
}  
  
QPushButton#pushButton_onlinemusic:pressed {  
    background-color: rgba(0, 255, 0, 0.5); /* 按下时亮绿色 */  
}</string>
   </property>
   <property name="text">
    <string>在线云音乐</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_AIgeneratedimage">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>211</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
